pkg/csi/service/wcpguest/controller.go

```diff
@@ -321,11 +345,32 @@ func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequ
 		} else {
 			attributes[common.AttributeDiskType] = common.DiskTypeBlockVolume
 		}
+		var accessibleTopology []*csi.Topology
+		if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA) {
+			// Retrieve the latest version of the PVC
+			pvc, err = c.supervisorClient.CoreV1().PersistentVolumeClaims(c.supervisorNamespace).Get(
+				ctx, supervisorPVCName, metav1.GetOptions{})
+			if err != nil {
+				msg := fmt.Sprintf("failed to get pvc with name: %s on namespace: %s from supervisorCluster. Error: %+v",
+					supervisorPVCName, c.supervisorNamespace, err)
+				log.Error(msg)
+				return nil, csifault.CSIInternalFault, status.Errorf(codes.Internal, msg)
+			}
+			topologyRequirement, err := generateVolumeAccessibilityRequirementsFromPVCAnnotation(pvc)
```

Comment:
Only set AccessibleTopology when the volume is requested using `AccessibilityRequirements` after checking feature gate